CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [FEATURE] AlertManager: Add support for Webex Receiver. #5493
```

Comment:
We also add `discord` and `telegram` in this pr. Can you update changelog?

Reply:
Done.